src/Microsoft.Restier.Core/ApiBase.cs

```diff
@@ -121,14 +124,31 @@ public void Dispose()
         }
 
         /// <summary>
+        /// Configure services for this API.
+        /// </summary>
+        /// <param name="builder">
+        /// The <see cref="ApiBuilder"/> with which to create an <see cref="ApiConfiguration"/>.
+        /// </param>
+        /// <returns>The <see cref="ApiBuilder"/>.</returns>
+        [CLSCompliant(false)]
+        protected virtual ApiBuilder ConfigureApiBuilder(ApiBuilder builder)
```

Comment:
Suggest a more concise name `ConfigureApi` because we are actually configuring and finally building an `Api` here.